libs/shared/mozlog/README.md

```diff
@@ -0,0 +1,7 @@
+# shared-mozlog
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
Would be useful to document why this exists, e.g. to provide a legacy compatible logger.